doc/user-guide.md

```diff
@@ -444,30 +444,30 @@ func (r *ReconcileMemcached) Reconcile(request reconcile.Request) (reconcile.Res
  		// TODO(user): Add the cleanup steps that the operator needs to do before the CR can be deleted
  		// Update finalizer to allow delete CR
  		memcached.SetFinalizers(nil)
- 
+
  		// Update CR
  		err := r.client.Update(context.TODO(), memcached)
  		if err != nil {
  			return reconcile.Result{}, err
  		}
  		return reconcile.Result{}, nil
  	}
- 	
+
  	// Add finalizer for this CR
  	if err := r.addFinalizer(reqLogger, instance); err != nil {
  		return reconcile.Result{}, err
  	}
  	...
- 
+
  	return reconcile.Result{}, nil
  }
- 
+
 //addFinalizer will add this attribute to the Memcached CR
 func (r *ReconcileMemcached) addFinalizer(reqLogger logr.Logger, m *cachev1alpha1.Memcached) error {
     if len(m.GetFinalizers()) < 1 && m.GetDeletionTimestamp() == nil {
         reqLogger.Info("Adding Finalizer for the Memcached")
         m.SetFinalizers([]string{"finalizer.cache.example.com"})
-    
+
```

Comment:
Given the more extensive changes to the finalizer example in #1524 (and since these are unrelated to this PR), want me to make sure I've got the spacing issues covered there?